src/__tests__/to-have-accessibility-state.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { View } from 'react-native';
+import { render } from '@testing-library/react-native';
+
+test('.toHaveAccessibilityState to handle explict state', () => {
+  const { getByTestId } = render(
+    <View>
+      <View testID="disabled" accessibilityState={{ disabled: true }} />
+      <View testID="selected" accessibilityState={{ selected: true }} />
+      <View testID="busy" accessibilityState={{ busy: true }} />
+      <View testID="checked-true" accessibilityState={{ checked: true }} />
+      <View testID="checked-mixed" accessibilityState={{ checked: 'mixed' }} />
+      <View testID="checked-false" accessibilityState={{ checked: false }} />
+      <View testID="expanded-true" accessibilityState={{ expanded: true }} />
+      <View testID="expanded-false" accessibilityState={{ expanded: false }} />
+
+      <View testID="disabled-selected" accessibilityState={{ disabled: true, selected: true }} />
+    </View>,
+  );
+
+  expect(getByTestId('disabled')).toHaveAccessibilityState({ disabled: true });
+  expect(getByTestId('disabled')).not.toHaveAccessibilityState({ disabled: false });
+  expect(() =>
+    expect(getByTestId('disabled')).toHaveAccessibilityState({ disabled: false }),
+  ).toThrow();
```

Comment:
is that not implied by the check above with `not.toHaveAccessibilityState({ disabled: false })`? (same for the other `expect(...).toThrow()`)

Reply:
Yes, it's the same but `expect(...).toThrow()` is there in order to achieve 100% code coverage. As otherwise Jest Matcher `message` property is not being evaluated ;-)

Reply:
oh good thing to know!